ui: add bar chart for node list memory & capacity

couchand opened the pull request:
Adds bar charts to capacity and memory columns on the node list.

![new-node-list](https://user-images.githubusercontent.com/793969/46502317-dccae700-c7f5-11e8-9c02-b3795137f10d.png)

Closes #29373

couchand committed:
ui: visually embed node lists in container

Release note: None

couchand committed:
ui: add bar chart for node list memory & capacity

Fixes: #29373
Release note (admin ui change): Adds a bar chart to the memory and
capacity usage columns on the node list.

couchand committed:
ui: sort memory & capacity columns by percent

Release note (admin ui change): Sorting by the memory or capacity
usage column on the node list now sorts by percentage used.

vilterp commented:
Seeing this in Chrome

![image](https://user-images.githubusercontent.com/7341/46504920-5adebc00-c7fd-11e8-8f54-0b3da2293857.png)

and this in Firefox

![image](https://user-images.githubusercontent.com/7341/46504936-6e8a2280-c7fd-11e8-8a4e-4e68f58b56f3.png)

Oy

vilterp commented on pkg/ui/src/views/cluster/containers/nodesOverview/barChart.tsx:
```diff
@@ -0,0 +1,113 @@
+  }
+
+  render() {
+    console.log("foobar!");
```
stray console.log

couchand replied:
foobar!

vilterp reviewed (commented):
One nit, as well as the FF vs Chrome issue. Otherwise LGTM.

couchand committed:
ui: fix node list bar height in Chrome

Chrome and Firefox have different rules about CSS selectivity
for SVG elements, apparently.  This is a hack, we should find
a simpler way to handle this (maybe changing the conflicting
classes).

Release note: None

couchand commented:
bors r+

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/30982)
<!-- Reviewable:end -->

couchand commented:
@vilterp I'm not seeing that in Chrome on Linux, I'll take a look Monday.

vilterp commented:
Hm looking ok now…
![image](https://user-images.githubusercontent.com/7341/46549836-4c49e080-c8a1-11e8-91bb-2c051521b77f.png)
Maybe it was a fluke, or only happens with single digits? Idk.

Overall this is looking really nice!
